vue-app/src/api/user.ts

```diff
@@ -4,16 +4,21 @@ import { Web3Provider } from '@ethersproject/providers'
 
 import { UserRegistry, ERC20 } from './abi'
 import { factory, ipfsGatewayUrl, provider } from './core'
+import { BrightIdError, getVerification } from './bright-id'
 
 export const LOGIN_MESSAGE = `Sign this message to access clr.fund at ${factory.address.toLowerCase()}.`
 
-// TODO update isVerified to isRegistered?
+export interface BrightId {
+  isLinked: boolean
+  isSponsored: boolean
+  isVerified: boolean // If is verified in BrightID
+}
 export interface User {
   walletAddress: string
   walletProvider: Web3Provider
   encryptionKey: string
-  isVerified: boolean | null // If is in user registry
```

Comment:
One note (that we can update in future PR) - some getters/components still rely on these properties, e.g. https://github.com/ethereum/clrfund/blob/develop/vue-app/src/components/CallToActionCard.vue

Reply:
Sure. I'll keep that in mind. We have a ticket for that specific component as well.